data/sql/updates/pending_db_world/rev_1605128047204479800.sql

```diff
@@ -0,0 +1,146 @@
+CREATE TEMPORARY TABLE IF NOT EXISTS `temp` select * from `dungeon_access_requirements` ORDER BY dungeon_access_id, faction;
+TRUNCATE `dungeon_access_requirements`;
+ALTER TABLE `dungeon_access_requirements`; 
+INSERT INTO `dungeon_access_requirements` SELECT * FROM `temp`;
+
+-- Verify result with this
+-- SELECT `id`, `quest_failed_text` FROM `access_requirement` WHERE `quest_failed_text` IS NOT NULL;
+
+
+-- Remove columns + rename table
+ALTER TABLE `access_requirement`
+DROP `item`,
+DROP `item2`,
+DROP `quest_done_A`,
+DROP `quest_done_H`,
+DROP `completed_achievement`,
+DROP `quest_failed_text`,
+RENAME TO `dungeon_access_template`,
+COMMENT='Dungeon/raid access template and single requirements';
+
+-- Rename columns
+ALTER TABLE `dungeon_access_template`
+CHANGE `mapId` `map_id` mediumint unsigned NOT NULL COMMENT 'Map ID from instance_template' AFTER `id`,
+CHANGE `difficulty` `difficulty` tinyint unsigned NOT NULL DEFAULT 0 COMMENT '5 man: 0 = normal, 1 = heroic, 2 = epic (not implemented) | 10 man: 0 = normal, 2 = heroic | 25 man: 1 = normal, 3 = heroic' AFTER `map_id`,
+CHANGE `level_min` `min_level` tinyint unsigned NULL DEFAULT NULL AFTER `difficulty`,
+CHANGE `level_max` `max_level` tinyint unsigned NULL DEFAULT NULL AFTER `min_level`,
+CHANGE `item_level` `min_avg_item_level` smallint unsigned NULL DEFAULT NULL COMMENT 'Min average ilvl required to enter' AFTER `max_level`,
+CHANGE `comment` `comment` varchar(255) COLLATE 'utf8_general_ci' NULL COMMENT 'Dungeon Name 5/10/25/40 man - Normal/Heroic' AFTER `min_avg_item_level`;
+
+-- Add KEY CONSTRAINTS
+ALTER TABLE `dungeon_access_template` ADD CONSTRAINT `FK_dungeon_access_template__instance_template` FOREIGN KEY (`map_id`) REFERENCES `instance_template` (`map`) ON DELETE CASCADE ON UPDATE CASCADE;
+ALTER TABLE `dungeon_access_requirements` ADD CONSTRAINT `FK_dungeon_access_requirements__dungeon_access_template` FOREIGN KEY (`dungeon_access_id`) REFERENCES `dungeon_access_template` (`id`) ON DELETE CASCADE ON UPDATE CASCADE;
+
+
+-- Add the acore_strings
+
+INSERT INTO `acore_string` (`entry`, `content_default`) VALUES (882, 'To enter, you must complete the following quest(s):');
+INSERT INTO `acore_string` (`entry`, `content_default`) VALUES (883, 'To enter, you must complete the following achievement(s):');
+INSERT INTO `acore_string` (`entry`, `content_default`) VALUES (884, 'To enter, you must have the following item(s) in your inventory:');
+INSERT INTO `acore_string` (`entry`, `content_default`) VALUES (885, '- Hint:');
+INSERT INTO `acore_string` (`entry`, `content_default`) VALUES (886, 'You cannot enter. Access requirements not met.');
+INSERT INTO `acore_string` (`entry`, `content_default`) VALUES (887, 'To be able to enter, your equipment\'s average item level must be superior or equal to %u. Your current equipment\'s average ilevel is: %u.');
+INSERT INTO `acore_string` (`entry`, `content_default`) VALUES (888, 'You must be below level %u to enter.');
```

Comment:
Always DELETE before INSERT, also you can do all this in one insert to limit the number of queries that need to be executed.

( I can confirm that none of these entries are currently in use in a default acore. good job! )